sycl/doc/UsersManual.md

```diff
@@ -390,6 +390,19 @@ and not recommended to use in production environment.
 
     NOTE: This flag is currently only supported with the CUDA and HIP targets.
 
+**`-fsycl-dump-device-code`** [DEPRECATED]
+    Enable dumping of device object files(SPIR-V and PTX files) during SYCL
+    offload compilation.
+    This option is replaced with -save-offload-code.
+
+    NOTE: This flag is currently only supported in
+    SYCL offloading to the CUDA and SPIR-V targets.
+
+ **`-save-offload-code`**
```

Comment:
Syntax of `-save-offload-code` and `-fsycl-dump-device-code` have an argument, that should also be reflected here.  Some thing like `-save-offload-code=<dir>` and `-fsycl-dump-device-code=<dir>`